src/program/lwaftr/tests/subcommands/loadtest_test.py

```diff
@@ -0,0 +1,60 @@
+"""
+Test the "snabb lwaftr loadtest" subcommand. Needs NIC names.
+
+Since there are only two NIC names available in snabb-bot, and we need to
+execute two programs networked to each other ("run" and "loadtest"), they
+are set to on-a-stick mode, so that they use one NIC each instead of two.
+"""
+
+import unittest
+
+from lib import sh
+from lib.test_env import BENCHDATA_DIR, DATA_DIR, SNABB_CMD, nic_names
+
+
+SNABB_PCI0, SNABB_PCI1 = nic_names()
+
+
+@unittest.skipUnless(SNABB_PCI0 and SNABB_PCI1, 'NICs not configured')
+class TestLoadtest(unittest.TestCase):
+
+    run_cmd_args = (
+        SNABB_CMD, 'lwaftr', 'run',
+        '--bench-file', '/dev/null',
+        '--conf', DATA_DIR / 'icmp_on_fail.conf',
+        # NUMA node #0.
+        '--cpu', '4',
```

Comment:
Possibly. We currently don't have env vars defined for the CPUs we run processes on.